backend/dal/Services/Concrete/ProjectService.cs

```diff
@@ -670,7 +674,7 @@ public Project SetStatus(Project project, Workflow workflow)
             // This could be extracted at some point to a configurable layer, but not required presently.
             switch (toStatus.Code)
             {
-                case ("SU"): // Submit
+                case ("RE"): // Review
```

Comment:
This is incorrect.  Review should not be a milestone and we only generate a project number when the request is submitted.